python/ray/data/dataset.py

```diff
@@ -2213,12 +2213,18 @@ def make_generator():
 
         return TorchIterableDataset(make_generator)
 
+    FeatureTypeSpec = Union[
```

Comment:
Nit: should we rename this to TensorflowFeatureTypeSpec as this is tensorflow specific?

Reply:
Fixed